### ISSUE TITLE
simple-ehm execution finishes with a SIGILL signal

### ISSUE THREAD

**Issue by pokemaniac05** [morrolinux/simple-ehm#2]
I'm running Ubuntu 20.10 amd64 with Linux kernel 5.8 and XFCE 4.14 on a PC with the following specifications:
* CPU Intel Pentium Dualcore 64bit
* RAM 2GB
I ran simple-ehm with python3.8-dgb and the following message was printed on stdout: 
`Illegal instruction (core dump created)`
Then I ran `gdb /usr/bin/python3.8-dgb` and I gave the following commands:
* `set args ./simple_ehm-runnable.py`
* `run`

`Program received signal SIGILL, Illegal instruction.
0x00007fffdc82ff10 in nsync::nsync_mu_init(nsync::nsync_mu_s_*) ()
   from /home/giuseppeniceforo/.local/lib/python3.8/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so`
* `bt`
I link the output [here](https://www.dropbox.com/s/78u51keob8wk03p/simple-ehm-runnable_python38-dgb-backtrace.txt?dl=0) because it is very long (564 lines).
I installed tensorflow with `pip install tensorflow` and the other dependencies with `pip install matplotlib`

**Comment by morrolinux** [comment]
Looks like the Tensorflow version your distro packages is compiled with some perhaps newer instructions which are unsupported on your CPU. Here's a link to a similar issue: https://github.com/tensorflow/tensorflow/issues/8976
The guy got around it compiling tensorflow from source

**Comment by pokemaniac05** [comment]
Thank you, I'll try to compile it manually from source

**Comment by pokemaniac05** [comment]
I tried to compile tensor flow three times, but the PC always crashed and I needed to reboot it with reset button. This PC can't compile TF and it is the only one that I can use to compile. May someone build TF for me and send the `.whl` package here? I need the `-mtune=pentium4` GCC option (Pentium 4 is very similar to my CPU) and no support for ROCm or CUDA (I use Intel HD Graphics). I don't need optional workspaces like Android's one.
